docs/knowledge_base/notifications.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Notifications
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+# Overview
+
+Phylum supports notification systems using email addresses or webhooks for various platforms including Slack, Microsoft Teams, and Discord. This document guides you through setting up notifications for each of these platforms.
+
+Notifications are a Phylum Pro feature and can be configured at the group or project level. Currently, the only trigger for notifications is [continuous monitoring](https://docs.phylum.io/docs/continuous_monitoring) events.
+
+## Notification Setup
+
+### Email Notifications
+
+1. Navigate to the 'Notifications' section in your Phylum group or project.
+2. Enter the email address where you wish to receive notifications.
+3. Save your settings to enable email notifications.
+
+### Slack Webhook Integration
+
+1. Create a Slack webhook following the guide at [Slack Webhook Setup](https://api.slack.com/messaging/webhooks#getting_started).
+2. In Phylum, navigate to the 'Notifications' section.
+3. Enter your Slack webhook URL.
+4. Perform a test to ensure the notifications are being sent correctly.
```

Comment:
They can't really do this. Since Continuous Monitoring is the only trigger, the only way to send a notification is for a new issue to be created that impacts one of their projects.

We could add a test notification endpoint, or automatically send out a test notification when their preferences are updated. But since it hasn't been done yet it shouldn't be in the docs.

Reply:
Gotcha, was wondering if we had a test that could be triggered. I'll update.